.buildkite/longruns/pipeline.yml

```diff
@@ -196,6 +196,39 @@ steps:
           slurm_mem_per_cpu: 16G
           slurm_reservation: "false"
 
+  - group: "CMIP simulations"
+
+    steps:
+      - label: "ClimaAtmos + ClimaLand + ClimaOcean + PrescribedSeaIce"
+        key: "cmip_target"
```

Comment:
I would prefer renaming this from `cmip_target` to something else, maybe just `cmip`. We have some different "target" runs in our CI, but our targets have changed over time so it loses its meaning as a label

Reply:
I called it `cmip_land`